src/components/ConfirmationPopUp.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div
+    v-if="isOpen"
+    class="modal fade fixed top-0 z-10 left-0 w-full h-full outline-none overflow-x-hidden overflow-y-auto"
+    id="exampleModalCenter"
+    tabindex="-1"
+    aria-labelledby="exampleModalCenterTitle"
+    aria-modal="true"
+    @click="closeModal"
+  >
+    <div
+      class="modal-dialog modal-dialog-centered relative w-auto pointer-events-none"
+    >
+      <Teleport to="body">
+        <div class="w-full h-full fixed top-0 bg-gray-500 opacity-90"></div
+      ></Teleport>
+
+      <div
+        class="modal-content border-none shadow-lg relative flex flex-col w-full pointer-events-auto bg-white bg-clip-padding rounded-md outline-none text-current"
+      >
+        <div
+          class="modal-header flex flex-shrink-0 items-center justify-between p-4 border-b border-gray-200 rounded-t-md"
+        >
+          <h5
+            class="text-xl font-medium leading-normal text-gray-800"
+            id="exampleModalScrollableLabel"
+          >
+            Confirmation PopUp
+          </h5>
+
+          <BaseButton
+            type="button"
+            class="btn-close box-content text-black border-none rounded-none opacity-50 focus:shadow-none focus:outline-none focus:opacity-100 hover:text-black hover:opacity-75 hover:no-underline"
+            data-bs-dismiss="modal"
```

Comment:
Why do you have bootstrap and tailwind installed at the same time? please choose one of them and remove the other since they both provide similar functionalities